templates/server.properties.erb

```diff
@@ -2,8 +2,8 @@
 #
 # See: http://kafka.apache.org/documentation.html#brokerconfigs for more details
 <%- @server_config.sort.map do |k,v|
-  if v.to_s.strip.length != 0 %>
+    if v.to_s.strip.length != 0 -%>
 <%= k %>=<%= v %>
```

Comment:
we can use `<%= k %>=<%= v %><%- unless v.to_s.strip.empty? -%>` and thus, we can get rid of the double end at the bottom.

or, we can use filter after before map